### DISCUSSION TITLE
Upgrade to Support new Mongodb Package & Laravel 10

### DISCUSSION THREAD

**Discussion by dave-propeller** [mostafamaklad/laravel-permission-mongodb#141] [General]
Hello,

MongoDB has recently taken over ownership of the jenseggers mongdb laravel package and version 4 is currently in ALPHA which includes laravel 10 support.

https://github.com/mongodb/laravel-mongodb

Just wondering if you had plans to keep this package up to date with those changes or will it continue to work only with jenseggers version 3.9?

Thanks in Advance.

**Comment by rmelet** [comment]
I have just create a pull request #153 with the required changes.
It fails because the test could not be run by scrutinizer.
Locally I also could not run the test because I had to run

`pecl install mongodb`

After updating the tests to use Monolog\Level instead of Monolog\Logger the tests run fine.
I think it also need to be installed in the scrutinizer image (docker?) but I am not familiar with it.
Could it have anything to do with php: pecl_extensions see: [https://scrutinizer-ci.com/docs/configuration/build_reference ](url)

Please take a look at it.
Thank you very much.

**Reply by rmelet** [reply]
I tried modifying the .scrutinizer.yml to add the mongodb extension.

Looks like it is working.
